### PR TITLE
Add cumberland council

### PR DESCRIPTION
**Pull request by pwae**
A copy of inner west - as it's using the same data provider.
Updated with relevant local examples.

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/cumberland_nsw_gov_au.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+TEST_CASES = {
+    "Berala Community Centre": {
+        "suburb": "Berala",
+        "street_name": "Woodburn Road",
+        "street_number": "98 to 104",
+    },
+    "McDonald's Auburn": {
+        "suburb": "Auburn",
+        "street_name": "Parramatta Road",
+        "street_number": "116",
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/cumberland_nsw_gov_au.py:19]: could you please provide a Test case with integer street_number (integer values should also be accepted) (and fix your code if it fails with integer value)

Comment (pwae) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/cumberland_nsw_gov_au.py:19]: Confirmed it works fine with an integer value, and added test case in:

Thanks!
```
./test_sources.py -s cumberland_nsw_gov_au                                                                                add-cumberland-council ✱
Testing source cumberland_nsw_gov_au ...
  found 104 entries for Berala Community Centre
  found 104 entries for McDonald's Auburn
  found 104 entries for Chickenlicious Guildford
```

**Comment by pwae** [comment]
Items resolved & branch bought up to date

**Comment by 5ila5** [comment]
Thanks for your contribution

**Pull request merged**